README.md

```diff
@@ -71,6 +71,12 @@ You can also post bug reports and feature requests in Github issues.
 
 
 ## Releases
+-[1.2.0](https://github.com/databricks/spark-deep-learning/releases/tag/v1.2.0) release: Spark 2.3.1, Python 3.6 & Scala 2.11 recommended. TensorFlow 1.10.0 required.
```

Comment:
If we already have release notes at https://github.com/databricks/spark-deep-learning/releases. We can remove this section and put a link to the release GitHub page.